_tour/tour-of-scala.md

```diff
@@ -48,14 +48,19 @@ Scala is equipped with an expressive type system that enforces statically that a
 
 ## Scala is extensible ##
 
-In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries. (In many cases, this can be done without using meta-programming facilities such as macros.)
+In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries.
 
-Scala is designed to interoperate well with the popular Java Runtime Environment (JRE). In particular, the interaction with the mainstream object-oriented Java programming language is as smooth as possible. Newer Java features like [annotations](annotations.html) and Java generics have direct analogues in Scala. Those Scala features without Java analogues, such as [default](default-parameter-values.html) and [named parameters](named-arguments.html), compile as close to Java as they can reasonably come. Scala has the same compilation model (separate compilation, dynamic class loading) like Java and allows access to thousands of existing high-quality libraries.
+In many cases, this can be done without using meta-programming facilities such as macros. For example,
 
-Please continue to the [next page](basics.html) in the Contents menu to read more.
+* [Implicit classes](http://docs.scala-lang.org/overviews/core/implicit-classes.html) allow adding extension methods to existing types.
+* [String interpolation](/overviews/core/string-interpolation.html) is user-extensible with custom interpolators.
+
+## Scala interoperates
+
+Scala is designed to interoperate well with the popular Java Runtime Environment (JRE). In particular, the interaction with the mainstream object-oriented Java programming language is as smooth as possible. Newer Java features like [annotations](annotations.html) and Java generics have direct analogues in Scala.
```

Comment:
"Newer" without "than" is relative, but SAM support is newer still. 

"Java generics" could maybe reference the specific tour page like the other mentions of features here.

Reply:
good feedback. followup commit added.